### ISSUE TITLE
MSLerror. This title is not available to watch instantly. 1044

### ISSUE THREAD

**Issue by Noctriss** [CastagnaIT/plugin.video.netflix#909]
#### Your Environment
- Netflix add-on version: 10.1 (Latest)
- Operating system version/name: Android
- Device model: n/a

Used Operating system:
* [x] Android
* [ ] iOS
* [ ] Linux
* [ ] OSX
* [ ] Raspberry-Pi
* [ ] Windows

### Describe the bug

After a fresh Kodi + Netflix plugin install I get the error 1044. MSLerror. THis title is not available to watch instantly on any video that I try. Log File: https://pastebin.com/mEzu14T4

#### Expected behavior
Normal playback of video

#### Actual behavior
Error is displayed and no video can be played

#### Steps to reproduce the behavior

1. Install Kodi
2. Install Netflix
3. Play Video

### Debug log

The debug log can be found from this link:

#### Installation
* [ ] I'm using other Netflix Repo 
* [ ] I'm using a different source 

#### Other information
If was working fine till about a week ago, and with no changes made it just stopped.

**Comment by CastagnaIT** [comment]
the only know solutions are in wiki (updated):
https://github.com/CastagnaIT/plugin.video.netflix/wiki/FAQ-%28Errors%29#this-title-is-not-available-to-watch-instantly

if you will find other solutions, please report here

**Comment by Noctriss** [comment]
Hello. Thank you for the link.

Unfortunately, I went into the device settings and it stated that "the device is no longer certified".

I guess that's the issue. Thanks again for the quick reply!

**Comment by occubr** [comment]
I have the same issue, i am using an raspberry pi 4 with libreelec 9.2.5 Leia and kodi 18.8. I am multibooting with berryboot from a external hdd.(libreelec, retropi, picroft and raspian) Could this bbe an issue? I don't know what else could be. hGO.ue for example works fine, youtube also, could somebody help me?

**Comment by Noctriss** [comment]
Everything else works for me as well.

I have some old android box that came with my internet / tv provider. I tried calling them about it and they openly told me that Netflix will not work (yey).

I managed to try and find a workaround for android devices by installing an old netflix .apk on it (v3.14 i think it is). It has bugs... but at least it works until I but a new android box and get rid of this thing they are calling "high tech" here (with android 4.1).
